### PR TITLE
Feature Events and News carusel in home page

### PR DESCRIPTION
**Pull request by alexkocharyan01**

### CODE REVIEW

File: src/components/EventsCarusel/EventsCarusel.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  const movePrev = () => {
+    if (currentIndex > 0) {
+      setCurrentIndex((prevState) => prevState - 1)
+    }
+  }
+
+  const moveNext = () => {
+    if (
+      carousel.current !== null &&
```
Comment (Arthur02V) [inline src/components/EventsCarusel/EventsCarusel.tsx:28]:
```suggestion
      carousel.current &&
```

File: src/components/EventsCarusel/EventsCarusel.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        path: `/events/${id}`,
+        params: '',
+        apply: true,
+      })
+    )
+  }
+
+  useEffect(() => {
+    if (carousel !== null && carousel.current !== null) {
+      carousel.current.scrollLeft = carousel.current.offsetWidth * currentIndex
```
Comment (Arthur02V) [inline src/components/EventsCarusel/EventsCarusel.tsx:47]:
```suggestion
    if (carousel && carousel.current) {
      carousel.current.scrollLeft = carousel.current.offsetWidth * currentIndex
```

File: src/components/EventsCarusel/EventsCarusel.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            <div className="mb-5 h-18">
+              <p className="text-lg text-black-light line-clamp-2 ">{item.header}</p>
+            </div>
+
+            <Button
+              size="fl"
+              variant="outlined"
+              onClick={() => redirectToIndividualNews(item.uuid)}
+            >
+              Read All
```
Comment (Arthur02V) [inline src/components/EventsCarusel/EventsCarusel.tsx:109]: Can we have a carousel logic in 1 component and make it reusable?

File: src/components/EventsCarusel/EventsCarusel.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      carousel.current.offsetWidth * currentIndex <= maxScrollWidth.current
+    ) {
+      setCurrentIndex((prevState) => prevState + 1)
+    }
+  }
+
+  const redirectToIndividualNews = (id: string) => {
+    dispatch(
+      viewsMiddleware.setRedirectionState({
+        path: `/events/${id}`,
```
Comment (Arthur02V) [inline src/components/EventsCarusel/EventsCarusel.tsx:38]: Typo in file name

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 3 above)
Comment (Arthur02V) [inline src/components/EventsCarusel/EventsCarusel.tsx:109]: Use translations

File: src/pages/index.tsx [hunk 5]
```diff
@@ -40,6 +43,24 @@ const Home = () => {
...
+  })
+
+  useEffect(() => {
+    Array(4)
+      .fill(1)
+      .map((x, y) => x + y)
+      .forEach((i) => {
+        dispatch(eventsMiddleware.fetchEventsList(i))
+      })
+  })
```
Comment (Arthur02V) [inline src/pages/index.tsx:62]: Doesn't seem right

**Comment by ArthurO2002** [comment]
Conflicts

File: src/pages/index.tsx (hunk 5 above)
Comment (alexkocharyan01) [reply src/pages/index.tsx]: done

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 3 above)
Comment (alexkocharyan01) [reply src/components/EventsCarusel/EventsCarusel.tsx]: done

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 4 above)
Comment (alexkocharyan01) [reply src/components/EventsCarusel/EventsCarusel.tsx]: done

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 3 above)
Comment (alexkocharyan01) [reply src/components/EventsCarusel/EventsCarusel.tsx]: done

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 2 above)
Comment (alexkocharyan01) [reply src/components/EventsCarusel/EventsCarusel.tsx]: done

File: src/components/EventsCarusel/EventsCarusel.tsx (hunk 1 above)
Comment (alexkocharyan01) [reply src/components/EventsCarusel/EventsCarusel.tsx]: done

**Pull request merged**